letsencrypt/plugins/manual.py

```diff
@@ -26,13 +26,17 @@
 class Authenticator(common.Plugin):
     """Manual Authenticator.
 
-    .. todo:: Support for `~.challenges.DVSNI`.
```

Comment:
So it appears there has been a little debate on this, but things like this are why I don't think we should be using the docstrings in `more_info`. It forces docstrings (which are primarily used by programmers) to be identical to the return value of `more_info` (which is seen by end users). There is no reason these things need to be coupled.

Personally, I'd just include whatever text you want shown to the user as a standalone string in `more_info`. If they're largely/completely duplicated with the docstring, so be it, but it allows us to more easily edit these strings in the future without unwanted side effects. Doing it this way, you can keep the `todo` comment.